src/main/java/org/opensha/sha/earthquake/faultSysSolution/ruptures/util/RupSetDiagnosticsPageGen.java

```diff
@@ -2632,9 +2632,10 @@ public static File plotRuptureHistogram(File outputDir, String prefix,
 		gp.getChartPanel().setSize(800, 600);
 		File pngFile = new File(outputDir, prefix+".png");
 		File pdfFile = new File(outputDir, prefix+".pdf");
+		File txtFile = new File(outputDir, prefix+".txt");
 		gp.saveAsPNG(pngFile.getAbsolutePath());
 		gp.saveAsPDF(pdfFile.getAbsolutePath());
-		
+        gp.saveAsTXT(txtFile.getAbsolutePath());
```

Comment:
@voj this indentation looks off in github

Reply:
the original lines have tabs :/

Reply:
tabs :-(